### PR TITLE
Hide Upload files button from file navigation in the repo

### PR DESCRIPTION
**Pull request by hkdobrev**
The uploading of files directly is nice, but the drag-n-drop ability is enough.

This of course could be merged a bit later once it has rolled out and people are familiar the upload feature is available. One could of course say users of this extension would not care about that feature.

### CODE REVIEW

File: extension/content.css [hunk 1]
```diff
@@ -158,3 +158,8 @@
 #dashboard .alert.git_hub {
 	display: none !important;
 }
+
+/* Remove Upload files button; drag and drop is available anyway*/
```
Comment (sindresorhus) [inline extension/content.css]: `Remove` => `remove` and `anyway*/` => `anyway */` for consistency

File: extension/content.css [hunk 2]
```diff
@@ -158,3 +158,8 @@
...
+.file-navigation a[href$="/upload"] {
+	display: none;
```
Comment (sindresorhus) [inline extension/content.css]:
```
display: none !important;
```

Just to be sure.

**Comment by sindresorhus** [comment]
:+1:

**Comment by hkdobrev** [comment]
@sindresorhus amended.

**Pull request merged**